### PR TITLE
New package: HIKARI-FIELD.HIKARI-FIELD-CLIENT version 1.1.8

### PR DESCRIPTION
**Pull request by xiSage**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157022)

### CODE REVIEW

**Comment by simulified** [comment]
There is a problem
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 1000; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 884.)

File: manifests/h/HIKARI-FIELD/HIKARI-FIELD-CLIENT/1.1.8/HIKARI-FIELD.HIKARI-FIELD-CLIENT.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+NestedInstallerFiles:
+- RelativeFilePath: HIKARI-FIELD-CLIENT-Setup-1.1.8.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-03-07
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/h/HIKARI-FIELD/HIKARI-FIELD-CLIENT/1.1.8/HIKARI-FIELD.HIKARI-FIELD-CLIENT.installer.yaml:17]: Hi @xiSage, 

Our `x86` validations are performed on 32-bit OSes. Based on the error above, this appears to be an `x64` application.

```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Verify: 
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**